lib/GetStream/StreamChat/Client.php

```diff
@@ -112,7 +112,7 @@ public function getBaseUrl()
         }
         $localPort = getenv('STREAM_LOCAL_API_PORT');
         if ($localPort) {
-            return "http://localhost:$localPort/api";
+            return "http://localhost:$localPort";
```

Comment:
why do we have this change?

Reply:
in order to run the integration tests locally